capstone-ui/src/components/ProfileHeader/ProfileHeader.jsx

```diff
@@ -77,6 +80,14 @@ export default function ProfileHeader({
       .catch(function (error) {
         alert(`Error! ${error.message}`);
       });
+
+    if (setShouldUpdate && typeof setShouldUpdate == "function") {
```

Comment:
why are you checking if setShouldUpdate is a function?

Reply:
I was checking because sometimes I don't pass setShouldUpdate as a props to this component, as ProfileHeader only needs to be updated in certain situations